app/assets/javascripts/discourse/app/components/group-selector.js

```diff
@@ -38,7 +38,9 @@ export default class GroupSelector extends Component {
       onChangeItems: (items) => {
         selectedGroups = items;
 
-        if (this.onChangeCallback) {
+        if (this.onChange) {
+          this.onChange(items.join(","));
+        } else if (this.onChangeCallback) {
```

Comment:
Had to add this `onChange` property for the right signature. This component doesn't seem to be used in any other place in any of our repos, but I kept the other property just in case.